crates/aptos-rosetta/src/lib.rs

```diff
@@ -80,12 +91,18 @@ pub fn bootstrap(
     chain_id: ChainId,
     api_config: ApiConfig,
     rest_client: Option<aptos_rest_client::Client>,
+    supported_currencies: HashSet<Currency>,
```

Comment:
To start is it just APT?

Reply:
do we expect other currencies? I'm trying to understand what we're reviewing for. Would be much easier if it's just APT.

Reply:
yes, it needs to support other currencies